CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## latest
```

Comment:
this project generates release notes from commit messages. No need to add manually.

Reply:
removed changes in this file